cg/meta/workflow/analysis.py

```diff
@@ -712,7 +712,7 @@ def get_data_analysis_type(self, case_id: str) -> str | None:
         analysis_types: set[str] = {
             link.sample.application_version.application.analysis_type for link in case.links
         }
-        if len(analysis_types) > 1:
```

Comment:
Won't this give an unclear return statement? It just pops the first analysis type in the list but if we have several we might need some extra logic